From 5fe91c831e500b584b301b024ac7c05c61f214e1 Mon Sep 17 00:00:00 2001
From: Artur Wiebe <artur@4wiebe.de>
Date: Wed, 15 Feb 2023 17:22:21 +0100
Subject: [PATCH] journalctl: fix --follow with --merge

---
 journalctl.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/journal/journalctl.c b/src/journal/journalctl.c
index 11de07f..75bfcf7 100644
--- a/src/journal/journalctl.c
+++ b/src/journal/journalctl.c
@@ -543,10 +543,6 @@ static int parse_argv(int argc, char *argv[]) {
                 case 'f':
                         arg_follow = true;
 
-                        arg_boot = true;
-                        arg_boot_id = SD_ID128_NULL;
-                        arg_boot_offset = 0;
-
                         break;
 
                 case 'o':
-- 
2.39.1

