From c27c36f792ff77817a89aadd4246ccc8c97ce3a6 Mon Sep 17 00:00:00 2001
From: Artur Wiebe <artur@4wiebe.de>
Date: Mon, 23 Dec 2024 16:28:56 +0100
Subject: [PATCH 2/4] direct import STLLoader

---
 examples/jsm/loaders/STLLoader.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/jsm/loaders/STLLoader.js b/examples/jsm/loaders/STLLoader.js
index b94b806..aed3a90 100644
--- a/examples/jsm/loaders/STLLoader.js
+++ b/examples/jsm/loaders/STLLoader.js
@@ -7,7 +7,7 @@ import {
 	Loader,
 	Vector3,
 	SRGBColorSpace
-} from 'three';
+} from './three.module.js';
 
 /**
  * Description: A THREE loader for STL ASCII files, as created by Solidworks and other CAD programs.
-- 
2.47.1

